### PR TITLE
add radio button widget

### LINKED ISSUE #3957: Add support for radio buttons in json schema forms
## Is your feature request related to a problem? Please describe.

We've used radio buttons in the schema twice now, and they aren't supported in mobile yet.

## Describe the solution you'd like

Support radio buttons

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #3957 

## Change summary

Added radio buttons
Tested with the ADR form which uses radios for string and boolean types. Haven't gone further than that.. is working for those 
Tried to make them look like the web version, so have tweaked the circle sizes and colours to make that look ok.

### CODE REVIEW

**Comment by mark-prins** [comment]
form

**Review by joshxg — APPROVED** [review]
Nice :)

Just the commented line which we could remove- also not sure if the focus control is doing anything?

**Comment by mark-prins** [comment]
Removed the focus, didn't seem to make any difference :)
Not sure about the other line that could be removed.

File: src/widgets/JSONForm/widgets/Radio.js [hunk 1]
```diff
@@ -1,10 +1,68 @@
...
-  console.log('-------------------------------------------');
-  return <Text style={{ borderWidth: 1, marginLeft: 10 }}>RadioWidget</Text>;
+import { APP_FONT_FAMILY } from '../../../globalStyles/fonts';
+import { GREY, SUSSOL_ORANGE } from '../../../globalStyles/colors';
+
+export const Radio = ({ options, value, disabled, readonly, onChange }) => {
+  const { enumOptions, enumDisabled } = options;
+  const row = options ? options.inline : false;
+
+  // return <RadioForm ref={ref} radio_props={radioProps} initial={0} onPress={onChange} />;
```
Comment (joshxg) [inline src/widgets/JSONForm/widgets/Radio.js:14]: oh- this one! hehe

Comment (mark-prins) [reply src/widgets/JSONForm/widgets/Radio.js]: Doh! b8bdcbc9

**Review by joshxg — APPROVED** [review]
👍

**Comment by joshxg** [comment]
Side note: With the ADR form on the hub needing a patient name entered- new `form_schema` for each? 🤣

**Pull request merged**